apps/proxy-worker/README.md

```diff
@@ -1,3 +1,239 @@
 # Cyrus Proxy Worker (Cloudflare Workers)
 
-Documentation coming soon.
+A Cloudflare Worker that serves as an edge proxy for the Cyrus Linear Agent, handling OAuth flows, webhooks, and token management using KV storage.
+
+## 🚀 Quick Setup
+
+### Automated Setup (Recommended)
+
+```bash
+# Make sure you're authenticated with Cloudflare
+npx wrangler login
```

Comment:
had a ton of issues with this on a remote machine. [this](https://github.com/cloudflare/workers-sdk/issues/2874#issuecomment-1460120004) ended up being the best path for me